packages/env-tests/src/shared/attestation.ts

```diff
@@ -264,3 +209,106 @@ export async function pollForMessagesAndCompleteAttestations(
     attestationsToComplete = await attestations.getActionableAttestations(identifier, account)
   }
 }
+
+// Use the supplied salt, or if none supplied, go to ODIS and retrieve a pepper
+export async function getIdentifierAndPepper(
+  kit: ContractKit,
+  context: string,
+  account: string,
+  phoneNumber: string,
+  salt: string | null
```

Comment:
Name this argument pepper then? (I know I probably also called this salt when I copied it)